recipes/nftables/nftables/link-with-ncurses.patch

```diff
@@ -0,0 +1,70 @@
+--- nftables-0.6/configure.ac.orig	2016-10-21 20:14:50.501289372 +0000
```

Comment:
please add the patch header stuff here, I don't known why we need to link with ncurses :)